app/scss/components/actions.scss

```diff
@@ -90,8 +90,12 @@ div.actions, td.actions > div[tabindex="0"] {
     }
   }
 
-  &:hover ul, ul:hover, &:focus ul, &:focus-within ul {
+  &:hover ul, ul:hover {
```

Comment:
Where did the focus styles go?

Reply:
Or in other words: did you try using this with a keyboard?

Reply:
Removing them doesnt seem to make a difference?
They have been readded

Reply:
It should add the same styles when tabbing around the interface to the ul as when hovering over it.  Only works if the ul is focusable or if the ul has elements within it that are focusable.